examples/list_transactions/list_transactions.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet/models/filter"
+
+	"examples"
+)
+
+func main() {
+	defer examples.HandlePanic()
+
+	const server = "http://localhost:3003/v1"
+
+	if examples.ExampleXPriv == "" {
+		fmt.Println(examples.ErrMessage("xPriv"))
+		os.Exit(1)
```

Comment:
Maybe we could put this code into the "utils.go" function.
And use it everywhere it's needed.

Reply:
created 3 new functions in utils.go for this